handlers.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+module Handlers
+  def list_books
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    @people.each { |person| puts person }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'That is not a valid input'
+      return
+    end
+
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.casecmp == 'y'
```

Comment:
- [ ] I got an error when I tried to create a student. I think it has to do with the case conversion.

| Error |
:-------:|
<img width="678" alt="Screen Shot 2022-11-04 at 9 01 13 PM" src="https://user-images.githubusercontent.com/39523312/200074671-6603e01a-bf02-4f98-ae2d-534e7650ebba.png"> |



- [ ]  If you are trying to get the lowercase for the input please use `.downcase` 

Example: 
```rbuy
parent_permission = gets.chomp.downcase == "y"
```

Reply:
yes i followed rubocop and changed to **casecmp .**